pingouin/pairwise.py

```diff
@@ -315,15 +316,21 @@ def pairwise_tests(
         assert all([between in data.keys(), within in data.keys()])
 
     # Create col_order
+    if parametric == True:
+        am, ae, bm, be = "mean(A)", "std(A)", "mean(B)", "std(B)"
```

Comment:
please use more explicit variable names

Reply:
am: --> desca mean that description of a
ae: --> stata mean that statistical info of a 
bm: --> desca mean that description of b
be: --> stata mean that statistical info of b